di_website/whatwedo/blocks.py

```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
 
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
```

Comment:
Could you please rename this to LocationBlock

Reply:
Just done and pushed